config/mongoose.js

```diff
@@ -1,15 +1,29 @@
 const mongoose = require('mongoose')
 
-mongoose
-  .connect(process.env.DATABASE_URL, {
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useUnifiedTopology: true,
-    useFindAndModify: false
-  })
-  .then(() => {
-    console.log('mongodb connection successful')
-  })
-  .catch((err) => {
-    console.log('mongodb connection error', err)
-  })
+class Database {
```

Comment:
Hey @ksraj123 The part which I have already done, no need to do that again.
We can discuss our approach

Reply:
Did that again through a different approach to implement singleton pattern, through I am not happy with it. The current approach is good enough but I think we would want our Database to be instantiable only inside the App class, looking for ways to implement that in javascript.